### PR TITLE
Add german translation

### PR DESCRIPTION
**Pull request by wrth1337**
Added the german translation for the readme-file.
Also implemented the linking to the different translations as mentioned in #106.

### CODE REVIEW

**Review by kyu08 — COMMENTED** [review]
Great work!
I couldn't add german translation without your PR! Thank you so much!

I left some comments. Please check it out.

File: doc/README-de.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+# 🛠️ Eigenschaften
+- Auswählen und Ausführen eines Make-Targets mit fzf
+- Unterstützt `include` directive
+- **(In Entwicklung)** Unterstützt Konfigurations-Dateien
+- **(In Entwicklung)** Command-Verlauf / History
+
+# 👓 Voraussetzungen
+- [bat](https://github.com/sharkdp/bat) (Für die Zukunft ist geplant, dass es auch mit `cat` funktioniert, aber derzeit funktioniert es nur mit `bat`.)
+
+# 📦 Installation
```
[the remark this answers is not in the document]
Comment (wrth1337) [reply doc/README-de.md:28]: Installation is already the german word for Installation ;)

File: doc/README-de.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+## nix
+Oder man kann `nix` verwenden, um eine Entwicklungsshell mit den Dependencies zu erstellen.
+
+Führe im Stammverzeichnis des Repo den folgenden Befehl aus:
+```nix
+nix develop
+```
+
+# 👥 Contribution
```
[the remark this answers is not in the document]
Comment (wrth1337) [reply doc/README-de.md:100]: I woudln't translate the word 'Contribution'. It's a common used word in german (esp. in the german github community).

File: doc/README-de.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- Contributions sind willkommen!
+- Wenn du eine Funktionsanfrage hast, erstelle bitte zuerst ein Issue.
+- Wenn du fzf-make zu einem Paketmanager hinzugefügt hast, lass es mich bitte wissen. (oder sende bitte einen PR, um die Installation über den Paketmanager in die `README.md` aufzunehmen)
+- Wenn Fragen bestehen, gerne einfach ein Issue erstellen und fragen.
+
+# 🗒 Related Article(s)
```
[the remark this answers is not in the document]
Comment (wrth1337) [reply doc/README-de.md]: Translated to "Verwandte Artikel".

**Review by kyu08 — APPROVED** [review]
Looks great!
Thank you so much!

**Pull request merged**